File: ASP.NETCoreWebApplication/Models/ViewModel/BudgetView.cs
using System;
using ASP.NETCoreWebApplication.Models.Enum;

namespace ASP.NETCoreWebApplication.Models.ViewModel
{
	public class BudgetView
	{
		public Int32 Id { get; set; }
		public DateTime CreatedDate { get; set; }
		public TypeOperation TypeOperation { get; set; }
		public CategoryTypeOperation CategoryTypeOperation { get; set; }
		public Decimal OperationSum { get; set; }
		public String Description { get; set; }

		public BudgetView() { }
		
		public BudgetView(Int32 id, DateTime createdDate, TypeOperation typeOperation, CategoryTypeOperation categoryTypeOperation, Decimal operationSum, String description)
		{
			Id = id;
			CreatedDate = createdDate;
			TypeOperation = typeOperation;
			CategoryTypeOperation = categoryTypeOperation;
			OperationSum = operationSum;
			Description = description;
		}
	}
}

File: ASP.NETCoreWebApplication/ClientApp/src/model/BudgetView.ts
import {TypeOperation} from "../enum/TypeOperation";
import {CategoryTypeOperation} from "../enum/CategoryTypeOperation";

export class BudgetView{
	public id: number;
	public createdDate: Date;
	public typeOperation: TypeOperation;
	public categoryTypeOperation: CategoryTypeOperation;
	public operationSum: number;
	public description: string;
	constructor(id: number,
				createdDate: Date,
				typeOperation: TypeOperation,
				categoryTypeOperation: CategoryTypeOperation,
				operationSum: number,
				description: string) {
		this.id = id;
		this.createdDate = createdDate;
		this.typeOperation = typeOperation;
		this.categoryTypeOperation = categoryTypeOperation;
		this.operationSum = operationSum;
		this.description = description;
	}
}